ipaserver/dnssec/_ods21.py

```diff
@@ -47,9 +47,9 @@ def get_keys_for_zone(self, zone_id):
             key['active'] = None
             key['retire'] = None
             key['dead'] = None
-            if row['keyType'] == 2:
+            if row['role'] == 2:
                 key['keytype'] = 256
-            elif row['keyType'] == 1:
+            elif row['role'] == 1:
```

Comment:
I don't know if it's worth mentioning, I know next to zero about DNSSEC, but the best documentation for the magic numbers 256 and 257 that I was able to dig up was at https://docs.infoblox.com/display/NAG8/DNSKEY+Resource+Records

The keyType -> role change checks out.